modules/standard/CommDiagnostics.chpl

```diff
@@ -18,7 +18,8 @@
  * limitations under the License.
  */
 
-/*
+/* Supports reasoning about Chapel code's communication requirements.
```

Comment:
To me "communication requirements" sounds like the things we require out of the hardware. I might say "Supports counting and reporting network communication operations."